test/WebJobs.Extensions.DurableTask.Analyzers.Test/ActivityFunction/NameAnalyzerTests.cs

```diff
@@ -55,31 +56,35 @@ public static class HelloSequence
         {
             // Matching names (strings)
 
-            await context.CallActivityAsync<string>(""Test_MatchingStrings"", ""Tokyo"");
+            await context.CallActivityAsync<string>(""Test_MatchingStrings"", ""Sandra Bland"");
```

Comment:
I'm not comfortable with these activity name changes. Let's stick with the city name convention.